HW03.md

```diff
@@ -0,0 +1,61 @@
+HW03
+================
+Mingyu Qi
+7/20/2020
+
+## Load libraries
+
+``` r
+library("ggplot2")
+```
+
+## Importing data set
+
+Import a CSV data set of daily snowfall from January to March in the 9
+US counties with **highest recorded daily snowfall** during 2010-2016
+period.
+
+``` r
+snow <- read.csv("D:/BSDG_30901/Project/HW03/Snow.csv", head=TRUE)
+attach(snow)
+```
+
+## Create a numeric variable based on factor variable “new date”
+
+``` r
+new_date <- factor(new_date, levels=unique(new_date[order(month,day)]), ordered=TRUE)
+new_date_2 <- as.numeric(new_date)
+```
+
+## Create a panel of line graphs of daily snowfall and z-score of daily snowfall by date
+
+``` r
+#Plot daily snowfall on date
+ggplot(snow, aes(new_date_2, daily_snow_county_inch)) +
+geom_line(aes(group=1, color="Daily Snowfall (Inch)", linetype="Daily Snowfall (Inch)"), size=1) +
+#Add another line: z-score of daily snowfall 
+geom_line(aes(new_date_2, daily_snow_county_z, group=1, color="Z-Score of Daily Snowfall", 
+              linetype = "Z-Score of Daily Snowfall"), size=1) +
+#Add annotation of max snowfall and max z-score within each group
+geom_label(x=45.5, y=33, aes(label = paste("Max Snowfall: ", max_snow, "Max Z-Score: ", max_z)), fill="orange", size=3) +
+#Create a panel based on variable group_label, which indicates state, county, and year
+facet_wrap(~group_label) + 

```

Comment:
Wow this is a very good way to create a panel of separate graphs through a variable, I will have to learn it myself too!

Reply:
I agree, it is a very clear way to create a panel for multiple plots.